content/community/lup-telegram.md

```diff
@@ -0,0 +1,8 @@
++++
+title = "Telegram"
+description = "Element is our preferred chat platform"
```

Comment:
description doesn't match up w Telegram

Reply:
Ya, I'd copy the element a bunch of times :sweat_smile: